guides/extensions/index.md

```diff
@@ -2,15 +2,15 @@ import DocCardList from '@theme/DocCardList';
 
-Currently, there are four packages in the extension repository. Example extension libraries include React Hooks for AFJ and a REST API wrapper.
+Currently, there are four packages in the extension repository. Example extension libraries include React Hooks for Credo and a REST API wrapper.
 
-| Package                                                                                                    | Version                                                                                                          | Description                                             |
-| ---------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
-| [`@aries-framework/redux-store`](https://www.npmjs.com/package/@aries-framework/redux-store)               | ![@aries-framework/redux-store version](https://img.shields.io/npm/v/@aries-framework/redux-store)               | Redux toolkit wrapper around Aries Framework JavaScript |
-| [`@aries-framework/rest`](https://www.npmjs.com/package/@aries-framework/rest)                             | ![@aries-framework/rest version](https://img.shields.io/npm/v/@aries-framework/rest)                             | Rest endpoint wrapper for using your agent over HTTP    |
-| [`@aries-framework/react-hooks`](https://www.npmjs.com/package/@aries-framework/react-hooks)               | ![@aries-framework/react-hooks version](https://img.shields.io/npm/v/@aries-framework/react-hooks)               | React Hooks for data handling and agent interaction     |
-| [`@aries-framework/push-notifications`](https://www.npmjs.com/package/@aries-framework/push-notifications) | ![@aries-framework/push-notifications version](https://img.shields.io/npm/v/@aries-framework/push-notifications) | Push notification plugin for Aries Framework JavaScript |
+| Package                                                                                                    | Version                                                                                                          | Description                                          |
+| ---------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| [`@aries-framework/redux-store`](https://www.npmjs.com/package/@aries-framework/redux-store)               | ![@aries-framework/redux-store version](https://img.shields.io/npm/v/@aries-framework/redux-store)               | Redux toolkit wrapper around Credo                   |
```

Comment:
I think we can also update the package names already . Or do you want to do it in a separate pr